libs/utils/src/http/endpoint.rs

```diff
@@ -129,7 +159,9 @@ pub fn make_router() -> RouterBuilder<hyper::Body, ApiError> {
         .middleware(Middleware::post_with_info(
             add_request_id_header_to_response,
         ))
-        .get("/metrics", prometheus_metrics_handler)
+        .get("/metrics", |r| {
+            RequestSpan(prometheus_metrics_handler).handle(r)
```

Comment:
do we want these as well ?

Reply:
It really depends on how we're planning to use spans: AFAIK, the default behavior of opentelemetry exporter includes such event-less spans, so you can store the data when every endpoint was accessed.

So, I'd leave it as is, since see no issues with that, but can remove if you have a reason.